src/Stratis.Bitcoin.Networks/StratisMain.cs

```diff
@@ -52,6 +52,7 @@ public StratisMain()
             this.DefaultConfigFilename = StratisDefaultConfigFilename;
             this.MaxTimeOffsetSeconds = 25 * 60;
             this.CoinTicker = "STRAT";
+            this.BanTimeSeconds = 16000; // 500 (MaxReorg) * 64 (TargetSpacing) / 2
```

Comment:
`// 500 (MaxReorg) * 64 (TargetSpacing) / 2 = 4 hours, 26 minutes and 40 seconds`

Reply:
Done.